Shortcuts/apps.json

```diff
@@ -991,6 +991,24 @@
         "appDetectionInfo": {
           "applicationIdentifier": "\\\\Todoist\\.exe$"
         }
+      },
+      {
+        "internalName": "Files",
+        "appDisplayName": "Files",
+        "logo16": "ms-appx:///Assets/Shortcuts/Icons/Square44x44Logo.targetsize-16.png",
+        "logo64": "ms-appx:///Assets/Shortcuts/Icons/Square44x44Logo.targetsize-64.png",
+        "searchKeywords": [
+          "File",
+          "Manager",
+          "Tabs",
+          "Columns",
+          "Preview"
+        ],
+        "shortcutsFile": "Utilities & Tools/files.json",
+        "appDetectionInfo": {
+          "applicationIdentifier": "files.exe",
```

Comment:
I would recommend to change this to `\\\\files\\.exe$`, which makes sure the path ends by `\files.exe`.

Reply:
Pushed 👍